packages/mdc-slider/foundation.ts

```diff
@@ -21,113 +21,123 @@
  * THE SOFTWARE.
  */
 
-import {cssClasses, strings, numbers} from './constants';
-import MDCSliderAdapter from './adapter';
-
-import {getCorrectEventName, getCorrectPropertyName} from '@material/animation/index.ts';
+import {getCorrectEventName, getCorrectPropertyName} from '@material/animation/index';
 import {MDCFoundation} from '@material/base/foundation';
+import {SpecificEventListener} from '@material/base/index';
+import {MDCSliderAdapter} from './adapter';
+import {cssClasses, numbers, strings} from './constants';
+
+type UpEventType = 'mouseup' | 'pointerup' | 'touchend';
```

Comment:
should we also be exporting these types?

Reply:
They're only used internally within the foundation; none of our public APIs use them. So I think we're probably safe keeping them private.